backend/src/models/tests/artist.test.ts

```diff
@@ -0,0 +1,31 @@
+import { Artist } from "../artist.model";
+import { database } from "../../config/database";
+
+beforeAll(async () => {
+  await Artist.sync({ force: true });
+});
+
+test("create artist", async () => {
+  expect.assertions(1);
+  const params = {
+    name: "testartist",
+  };
+  const artist = await Artist.create<Artist>(params).catch((err: Error) =>
+    fail("Creating artist failed.")
+  );
+  expect(artist.id).toEqual(1);
+});
+
+test("get artist", async () => {
+  expect.assertions(3);
+  const artist = await Artist.findByPk(1);
+  expect(artist).not.toEqual(null);
+  expect(artist!.id).toEqual(1);
+  expect(artist!.name).toEqual("testartist");
+});
```

Comment:
I thought unit tests were supposed to be written independently from other tests but this one seems to rely on the previous one... Is that okay?

Reply:
yes you are right, I was just trying to test as much stuff as possible by the meeting today, but that is best practice, ill change it before merge

Reply:
changed it to basically clear the database before every test using beforeEach()